xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -19,8 +19,10 @@
 
 #ifdef BUILD_KODI_ADDON
 #include "../DemuxPacket.h"
+#include "../InputStreamConstants.h"
```

Comment:
Is this correct?

Reply:
Yes, the headers from videoplayer becomes copied to kodi's base include dir by install (from addon then `#include <kodi/InputStreamConstants.h>`.